custom-connectors/PixelnebulaFlare/readme.md

```diff
@@ -0,0 +1,15 @@
+# Pixelnebula Flare Connector
```

Comment:
It'd be great to have a description here that goes into detail on the value the connector offers and what it does (on a high level).